addon-test-support/should-use-middleware-reporter.ts

```diff
@@ -0,0 +1,10 @@
+import { ENABLE_A11Y_MIDDLEWARE_REPORTER } from './cli-options';
```

Comment:
If we rename this to `useMiddlewareReporter`, should we also rename this const to `USE_A11Y_MIDDLEWARE_REPORTER`?

Reply:
I'd rather keep it as is so it's consistent with `ENABLE_A11Y_AUDIT` and `enableA11yAudit=true`.